src/pages/en/guides/images.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Images
+description: Learn how to use images in Astro.
+i18nReady: true
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+Astro provides several ways for you to use images inside of your projects.
+
+### In `.astro` files
+
+Astro uses standard HTML `<img>` or JSX `<img />` elements to display images within your `.astro` files. 
```

Comment:
> Astro uses standard HTML <img> or JSX <img /> elements

Personal taste perhaps, but I don't think it's needed to say "the HTML or JSX img tag". We generally align ourselves toward being HTML first

Reply:
Noted, thanks! I can change the wording to just say HTML. I just know that if I DON'T put both tags, with and without the closing slash, someone's gonna PR it in there because "it works." 😅